src/gcloud_common_utils/blob_helper_local.py

```diff
@@ -2,7 +2,6 @@
 import glob
 import logging
 from typing import Set
-
```

Comment:
This empty line was here to indicate that typegaurd is a package from pypi and the other three are deafult python packages!